bspacks/refmtJsApi.ml

```diff
@@ -1,3 +1,9 @@
+(*
+ * Note: This file is currently broken, since Reason removed
+ * Refmt_api.Reason_syntax_util.Error in favor of Reerror's `Printexc.to_string e`
+*)
+open Js_of_ocaml
+
```

Comment:
what's the rational of opening such module

Reply:
I added this to get the missing JSOO functions linked used inside this script. This file is not used for any build artifacts used by our playground. I think I will undo this open just for the sake of not messing with the file

Reply:
dropped the line to be 100% consistent with the original file